main.js

```diff
@@ -0,0 +1,157 @@
+
+const BASE_URL = 'http://wttr.in/';
+const form = document.querySelector('form');
+const currentWeather = document.querySelector('article');
+const weatherIcon = document.createElement('img');
+
+form.addEventListener('submit', (event) => {
+    event.preventDefault();
+    document.querySelector('main p').hidden = true;
+
+    let city = event.target.location.value;
+    event.target.location.value = "";
+
+    fetch(`${BASE_URL}${city}?format=j1`)
+    .then((response)=> {
+        return response.json();
+
+    })
+    .then((json) => {
+        let feelsLikeTemp = getWeatherReport(currentWeather, json, city);
+        const ul = document.querySelector('ul');
+        const searchHistory = document.createElement('li');
+        let a = document.createElement('a');
+        a.textContent = city;
+        a.href = `${BASE_URL}${city}?format=j1`;
```

Comment:
making the `href` the api url is making it so that when the link its clicked we get the data object instead of making a fetch call. See photo 
<img width="959" alt="Screenshot 2023-01-13 at 10 34 35 AM" src="https://user-images.githubusercontent.com/71154265/212358495-1907b4b6-e2ce-425f-a996-2d8fcece3ece.png">


What you want is that when a user clicks that link a fetch call is made. You can do this with an event listener.